infraestructure/config/config.go

```diff
@@ -0,0 +1,47 @@
+package config
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+type Configuration interface {
+	Load() error
+}
+
+type configuration struct {
+	ConfData confData
+	Wd       string
```

Comment:
Initialisms should be all underscored or all caps

Same applies for other structs
```suggestion
	WD       string
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Done